mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -226,4 +282,10 @@ def save(context, name):
         Saves the current playlist to ``NAME.m3u`` in the playlist
         directory.
     """
-    raise exceptions.MpdNotImplemented  # TODO
+    uri = context.lookup_playlist_uri_from_name(name)
+    playlist = uri is not None and context.core.playlists.lookup(uri).get()
+    if not playlist:
+        playlist = context.core.playlists.create(name).get()
```

Comment:
This one is tricky, since there might be a mix of uri types in the tracklist, and we need the playlist to be able to store all of them I'm not sure how to get this working reliably.